eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -70,12 +70,15 @@ function GetAdjustedReadmeContent($pkgInfo, $lang){
     }
 
     $fileContent = $pkgInfo.ReadmeContent
```

Comment:
Doesn't look like we need to set the fileContent here any more

Reply:
Actually I take that back as there are some assumptions but not all of them assume this. We should keep this.

Reply:
It is working like fallback. If no title matches, then we still have readme to populate